src/ui-extensions.js

```diff
@@ -101,30 +105,77 @@ function previewQuery(id) {
 
   const results = getQueryResult(client, id);
 
-  const sheet = spreadSheet.insertSheet('preview');
+  const sheet = spreadSheet.insertSheet(`preview id ${id}`);
   fillDataAndChart(sheet, results.data, results.name);
 }
 
+/**
+ * Processes user input, stores it, and initializes a Google Sheet document
+ * @param {Array.<string>} values - Array containing username,
+ * password and metabaseUrl
+ */
 /* eslint-disable no-unused-vars */
+// eslint-disable-next-line require-jsdoc
+function processCredentials(values) {
+  storeUserInput(values);
+  insertDataIntoSheet();
+}
+
 /**
  * Stores user credentials in user properties for future use.
- *  @param {Array.<string>} values - Array containing username, password
+ * @param {Array.<string>} values - Array containing username,
+ * password and metabaseUrl
  */
-function processCredentials(values) {
-  /* eslint-enable no-unused-vars */
+function storeUserInput(values) {
+  /* eslint-disable no-unused-vars */
```

Comment:
Doesn't actually need a linting hint, because the function is used in `processCredentials`
```suggestion
```